plopfile.js

```diff
@@ -0,0 +1,73 @@
+module.exports = function (plop) {
+  plop.setGenerator("component", {
+    description: "Create a reusable component",
+    prompts: [
+      {
+        type: "input",
+        name: "name",
+        message: "What is the component name?",
+      },
+    ],
+    actions: [
+      {
+        type: "add",
+        path:
+          "packages/components/src/{{pascalCase name}}/{{pascalCase name}}.tsx",
+        templateFile: "plop-templates/Component/Component.tsx.hbs",
+      },
+      {
+        type: "add",
+        path:
+          "packages/components/src/{{pascalCase name}}/{{pascalCase name}}.module.css",
+        templateFile: "plop-templates/Component/Component.module.css.hbs",
+      },
```

Comment:
This section is the only change I made from the original PR https://github.com/chanzuckerberg/edu-design-system/pull/309